addons/sale_timesheet/static/tests/tours/sale_timesheet_tour.js

```diff
@@ -294,8 +294,8 @@ tour.register('sale_timesheet_tour', {
     content: 'Check the user sees Total Sold section',
     run: function () {},
 }, {
-    trigger: ".oe_button_box .o_stat_text:contains('Sales Order')",
-    content: 'Check the user sees Sales Order Stat Button',
+    trigger: ".oe_button_box .o_stat_text:contains('Sales Orders')",
+    content: 'Check the user sees x Sales Order Stat Button',
```

Comment:
Was this intended or is it a typo?
```suggestion
    content: 'Check the user sees Sales Order Stat Button',
```

Reply:
It was intended, as I was asked to create a new "x Sales Orders" stat button but I will remove it as I guess it makes more sense.